analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -96,11 +96,35 @@ class Bower(
                 }
             }
 
+        /**
+         * Parse information about the author. According to https://github.com/bower/spec/blob/master/json.md#authors,
+         * there are two formats to specify the authors of a package (similar to NPM). The difference is that the
+         * strings or objects are inside an array.
+         */
+        private fun extractAuthors(node: JsonNode): SortedSet<String> =
```

Comment:
To align with other similar functions, this should probably be named `parseAuthrors`.